process/common.go

```diff
@@ -711,6 +712,24 @@ func GetSortedStorageUpdates(account *vmcommon.OutputAccount) []*vmcommon.Storag
 	return storageUpdates
 }
 
+func CreateHeader(shardId uint32, marshalizer marshal.Marshalizer, headerBuffer []byte) (data.HeaderHandler, error) {
```

Comment:
maybe rename to `UnmarshalHeader` / `UnmarshalMetaheader` / `UnmarshalShardHeader` ?

Reply:
`Create` makes me think of a new header, created by constructor

Reply:
Right :) renamed.